sdk/helper/ldaputil/client.go

```diff
@@ -77,7 +83,11 @@ func (c *Client) DialLDAP(cfg *ConfigEntry) (Connection, error) {
 			if err != nil {
 				break
 			}
-			conn, err = c.LDAP.DialTLS("tcp", net.JoinHostPort(host, port), tlsConfig)
+			opt := ldap.DialWithTLSDialer(tlsConfig, &dialer)
+			conn, err = c.LDAP.Dial(net.JoinHostPort(host, port), opt)
```

Comment:
Is `net.JoinHostPort(host, port), opt` sufficient? Or is the URL scheme `ldap` also needed?

Reply:
It's not, I had forgot to commit the change with the scheme. Fixed and the tests should pass now.